tests/fixtures/umls_test_fixture.json

```diff
@@ -1 +1 @@
ethylphenyltetrahydropyridine", "Methylphenyltetrahydropyridine (substance)", "1-Methyl-4-phenyl-1,2,3,6-tetrahydropyridine [Chemical/Ingredient]", "1-Methyl-4-Phenyl-1,2,3,6-Tetrahydropyridine (MPTP)"], "types": ["T109", "T131"], "canonical_name": "1-Methyl-4-phenyl-1,2,3,6-tetrahydropyridine", "definition": "A dopaminergic neurotoxic compound which produces irreversible clinical, chemical, and pathological alterations that mimic those found in Parkinson disease."}, {"concept_id": "C0000098", "aliases": ["1-Methyl-4-phenylpyridinium", "1 Methyl 4 phenylpyridinium", "1-Methyl-4-phenylpyridinium Ion", "1-Methyl-4-phenylpyridinium Ion", "1 Methyl 4 phenylpyridinium Ion", "Cyperquat", "Cyperquat", "MPP+", "N-Methyl-4-phenylpyridine", "N-Methyl-4-phenylpyridine", "N Methyl 4 phenylpyridine", "1-Methyl-4-phenylpyridine", "1-Methyl-4-phenylpyridine", "1 Methyl 4 phenylpyridine", "N-Methyl-4-phenylpyridinium", "N-Methyl-4-phenylpyridinium", "Pyridinium, 1-methyl-4-phenyl-", "Pyridinium, 1-methyl-4-phenyl-", "1-Methyl-4-phenylpyridinium [Chemical/Ingredient]"], "types": ["T109", "T131"], "canonical_name": "1-Methyl-4-phenylpyridinium", "definition": "An active neurotoxic metabolite of 1-METHYL-4-PHENYL-1,2,3,6-TETRAHYDROPYRIDINE. The compound reduces dopamine levels, inhibits the biosynthesis of catecholamines, depletes cardiac norepinephrine and inactivates tyrosine hydroxylase. These and other toxic effects lead to cessation of oxidative phosphorylation, ATP depletion, and cell death. The compound, which is related to PARAQUAT, has also been used as an herbicide."}, {"concept_id": "C0000102", "aliases": ["1-Naphthylamine", "1-naphthylamine", "1 Naphthylamine", "alpha-Naphthylamine", "alpha-Naphthylamine", "alpha-naphthylamine", "alpha Naphthylamine", "Naphthalidine", "Naphthalidine", "8-Aminonaphthalene", "8-Aminonaphthalene", "8 Aminonaphthalene", "1-Aminonaphthalene", "1-Aminonaphthalene", "1 Aminonaphthalene", "1-Naphthalenamine", "1-Naphthalenamine", "1-Naththylamine", "1-Naththylamine", "a- Naphthylamine", "a- Naphthylamine", "a-Naphthylamine", "1-Naththylamine (substance)", "1-Naphthylamine [Chemical/Ingredient]"], "types": ["T109", "T131"], "canonical_name": "1-Naphthylamine", "definition": "A suspected industrial carcinogen (and listed as such by OSHA). Its N-hydroxy metabolite is strongly carcinogenic and mutagenic."}, {"concept_id": "C0000103", "aliases": ["1-Naphthylisothiocyanate", "1 Naphthylisothiocyanate", "alpha-Naphthylisothiocyanate", "alpha-Naphthylisothiocyanate", "alpha Naphthylisothiocyanate", "Naphthalene, 1-isothiocyanato-", "Naphthalene, 1-isothiocyanato-", "1-Naphthylisothiocyanate [Chemical/Ingredient]"], "types": ["T109", "T130", "T131"], "canonical_name": "1-Naphthylisothiocyanate", "definition": "A tool for the study of liver damage which causes bile stasis and hyperbilirubinemia acutely and bile duct hyperplasia and biliary cirrhosis chronically, with changes in hepatocyte function. It may cause skin and kidney damage."}]
```

Comment:
Apologies for this diff, I just hacked in an entity that was similar-ish to another one so that I could test for the definition threshold thing.